deploy/addons/storage-provisioner/storage-provisioner.yaml

```diff
@@ -25,4 +25,13 @@ spec:
   containers:
   - name: storage-provisioner
     image: gcr.io/k8s-minikube/storage-provisioner:v1.8.1
+    command: ["/storage-provisioner"]
     imagePullPolicy: IfNotPresent
+    volumeMounts:
+    - mountPath: /tmp
+      name: tmp
+  volumes:
+  - name: tmp
+    hostPath:
+      path: /tmp
+      type: Directory
```

Comment:
newline

Reply:
We also need to release it?
Merge this, then build and push container, and send a PR to bump the version?